conf/carbon.properties.template

```diff
@@ -85,7 +85,6 @@ carbon.major.compaction.size=1024
 ##Min max feature is added to enhance query performance. To disable this feature, make it false.
 #carbon.enableMinMax=true
 
-######## Global Dictionary Configurations ########
```

Comment:
**please search "gobaldict" and "global dict", some more places need to handle I guess**

Reply:
The dictionary read path is not removed, I will do it in another PR